src/components/backend-ai-session-list.ts

```diff
@@ -3905,15 +3914,8 @@ ${rowData.item[this.sessionNameField]}</pre
   agentListRenderer(root, column?, rowData?) {
     render(
       // language=HTML
-      // FIXME: temporally show allocated agent only, not in session-agent pair
       html`
-        <div class="layout vertical">
-          ${[...new Set(rowData.item.agents)]?.map(
-            (agent) => html`
-              <span>${agent}</span>
-            `,
-          )}
-        </div>
+        <pre>${rowData.item.agents_ids_with_container_ids?.join('\n')}</pre>
```

Comment:
@agatha197  I changed this line because It's better to handle the formatting logic on the UI side.